application/email/v0/config/tasks.json

```diff
@@ -0,0 +1,471 @@
+{
+  "$defs": {
+    "search-subject-text": {
+      "description": "Search for emails with a specific subject",
+      "instillAcceptFormats": [
+        "string"
+      ],
+      "instillUpstreamTypes": [
+        "value",
+        "reference",
+        "template"
+      ],
+      "instillUIOrder": 4,
+      "title": "Search Subject",
+      "type": "string"
+    },
+    "search-from-email": {
```

Comment:
`search-from-email` looks a little bit weird to me. How about we use `search-from` or `search-email-from` instead?